src/data-types.js

```diff
@@ -87,6 +87,18 @@ export function getTypes(options) {
 		}
 	}
 
+	var int64 = {
+		unpack: function(buffer){
+			var value = buffer.readUInt64LE(buffer.pointer);
```

Comment:
When I ran 'npm install' it finished without error. I admit I did no additional testing of any kind.